.eslintrc.js

```diff
@@ -1,139 +1,94 @@
-    'no-unused-vars': 'off', // prefer @typescript-eslint/no-unused-vars
-    '@typescript-eslint/no-unused-vars': [
-      'warn', {
-        'vars': 'all',
-        'args': 'all',
-        'argsIgnorePattern': '^_|^props',
-        'ignoreRestSiblings': true,
-      }
-    ],
-    'no-whitespace-before-property': 'warn',
-    'nonblock-statement-body-position': 'warn',
-    'object-curly-newline': ['warn', { 'multiline': true, 'consistent': true }],
-    'object-curly-spacing': ['warn', 'always'],
-    'one-var': ['warn', 'never'],
-    'operator-linebreak': ['warn', 'after'],
-    'padded-blocks': ['warn', 'never'],
-    'quote-props': ['warn', 'as-needed'],
-    'quotes': ['warn', 'single', { 'avoidEscape': true }],
-    'semi': ['warn', 'never'],
-    'space-before-blocks': 'warn',
-    'space-before-function-paren': ['warn', { 'anonymous': 'never', 'named': 'never', 'asyncArrow': 'always' }],
-    'space-in-parens': 'warn',
-    'space-infix-ops': 'warn',
-
-    // ES6
-    'arrow-parens': ['warn', 'as-needed'],
-    'arrow-spacing': 'warn',
-    'no-duplicate-imports': 'warn',
-    'no-restricted-syntax': [
-      'warn',
-      { 'selector': 'TSEnumDeclaration', 'message': 'Use a union of literals instead of an enum' },
-      { 'selector': 'JSXElement', 'message': 'Use react-hyperscript-helpers instead of JSX' }
-    ],
-    // TODO: Set 'variables' to 'true' after fixing the existing issues
-    'no-use-before-define': ['warn', { 'functions': true, 'classes': true, 'variables': false }],
-    'no-useless-rename': 'warn',
-    'no-var': 'warn',
-    'object-shorthand': 'warn',
-    'prefer-arrow-callback': 'warn',
-    'prefer-const': 'warn',
-    'prefer-template': 'warn',
-    'prefer-rest-params': 'warn',
-    'prefer-spread': 'warn',
-    'rest-spread-spacing': 'warn',
-    'template-curly-spacing': 'warn',
-    'vars-on-top': 'warn',
-
+  env: {
+    jest: true,
+  },
+  rules: {
+    indent: 'off',
+    'comma-dangle': 'off',
+    'import/prefer-default-export': 'off',
 
-    'lodash-fp/consistent-name': ['warn', '_'],
-    'lodash-fp/no-argumentless-calls': 'warn',
-    'lodash-fp/no-chain': 'warn',
-    'lodash-fp/no-extraneous-args': 'warn',
-    'lodash-fp/no-extraneous-function-wrapping': 'warn',
-    'lodash-fp/no-extraneous-iteratee-args': 'warn',
-    // 'lodash-fp/no-extraneous-partials': 'warn', // available in eslint-plugin-lodash-fp 2.2.0+
-    'lodash-fp/no-partial-of-curried': 'warn',
-    'lodash-fp/no-single-composition': 'warn',
-    'lodash-fp/no-submodule-destructuring': 'warn',
-    'lodash-fp/no-unused-result': 'warn',
-    'lodash-fp/prefer-compact': 'warn',
-    'lodash-fp/prefer-composition-grouping': 'warn',
-    'lodash-fp/prefer-flat-map': 'warn',
-    'lodash-fp/prefer-get': 'warn',
-    'lodash-fp/prefer-identity': ['warn', { 'arrowFunctions': false }],
-    'lodash-fp/preferred-alias': 'warn',
-    'lodash-fp/use-fp': 'warn',
+    // TODO: ticket all of these to be fixed or considered
```

Comment:
I don't think we necessarily want to enable _all_ these rules (for example, we'd intentionally disable `react/destructuring-assignment`). Maybe this comment could be something like "TODO: enable each of these rules or explain why they are disabled".